src/components/snackbar/Snackbar.tsx

```diff
@@ -37,24 +38,46 @@ export type SnackbarProps = {
   customVisibleDurationMS?: number;
 };
 
-export const Snackbar = ({
+// if position is ever toggled, it starts using two SnackbarInstances, one for top and one for bottom
+// one of them is always disabled
+export const Snackbar = (snackbarProps: SnackbarProps) => (
+  <>
+    {SNACKBAR_POSITIONS.map((position) => (
+      <SnackbarInstance
+        key={position}
+        {...snackbarProps}
+        position={position}
+        isDisabled={position !== (snackbarProps.position || 'top')}
+      />
+    ))}
+  </>
+);
+
+type SnackbarInstanceProps = SnackbarProps & {
+  /** setting this true moves the SnackbarInstance to the hidden position */
+  isDisabled: boolean;
+};
+
+const SnackbarInstance = ({
   textContent,
   position = 'top',
   actionButton,
   isDismissable,
   customVisibleDurationMS,
-}: SnackbarProps) => {
+  isDisabled,
+}: SnackbarInstanceProps) => {
   const styles = useStyles();
   const {t} = useTranslation();
 
-  const stableTextContent = useStableValue(textContent); // avoid triggering useEffects if no text has been changed
+  const stableTextContent = useStableValue(textContent, isDisabled); // avoid triggering useEffects if no text has been changed
```

Comment:
Tried replacing `!stableTextContent` with `(!stableTextContent || isDisabled)` instead, which seems to work, but caused a render "blink". Locking `textContent` while `isDisabled` doesn't have this problem.